Gemfile

```diff
@@ -2,9 +2,11 @@
 
 source "https://rubygems.org"
 
-gem "pry-byebug", require: false
-gem "rubocop", require: false
-gem "rubocop-shopify", ">=2.8.0", require: false
-gem "rubocop-performance", require: false
+group :development, :test do
```

Comment:
maybe not necessary? I'm not sure how these work for RubyGems so please enlighten me if you know :)

Reply:
so i should remove this?